app/src/org/commcare/dalvik/preferences/DeveloperPreferences.java

```diff
@@ -95,12 +96,23 @@ public static boolean isNewNavEnabled() {
         SharedPreferences properties = CommCareApplication._().getCurrentApp().getAppPreferences();
         return properties.getString(NAV_UI_ENABLED, CommCarePreferences.NO).equals(CommCarePreferences.YES);
     }
+    
+    public static boolean isCssEnabled(){
+        try{
+            SharedPreferences properties = CommCareApplication._().getCurrentApp().getAppPreferences();
+            return properties.getString(CSS_ENABLED, BuildConfig.DEBUG ? CommCarePreferences.YES : CommCarePreferences.NO).equals(CommCarePreferences.YES);
```

Comment:
I don't think we want Debug to make this true by default, it'll make the dev apps and the normal apps look very different by default